### ISSUE TITLE
Create an image dimension validation rule

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#88]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To set constraints on dimensions of images uploaded in the Media library uploader form component (#67).

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via eb23252f
